core/src/com/unciv/UncivGame.kt

```diff
@@ -24,8 +24,12 @@ import kotlin.concurrent.thread
 
 class UncivGame(
         val version: String,
-        private val crashReportSender: CrashReportSender? = null
+        private val crashReportSender: CrashReportSender? = null,
+        val exitEvent: ()->Unit = {}
```

Comment:
actually more Kotlin idiomatic to make in nullable and null as default
because in current situation we have empty func which will run every time